README.md

```diff
@@ -5,6 +5,16 @@
 This library provides instrumentation for Spring  Web applications (Boot and MVC). It creates tracing data for 
 server requests and also client requests (`RestTemplate` and `AsyncRestTemplate`).
 
+## Comparison to `spring-cloud-sleuth`
+This project provides instrumentation only for `spring-web` package. In other words it traces only
+HTTP requests made to Spring Boot/WEB app and outgoing requests using Spring RestTemplate. However it allows
+you to use OpenTracing API directly in your code and combine different OpenTracing instrumentations together
+very easy (e.g. [OpenFeign](https://github.com/OpenFeign/feign-opentracing)).
+
+Whereas [spring-cloud-sleuth](https://github.com/spring-cloud/spring-cloud-sleuth) combines
+instrumentations for different frameworks together and you are not allowed to use OpenTracing API or wire
```

Comment:
Full stop after "together". And then possibly "It is not currently possible to use the OpenTracing API (and therefore switch between tracing backends), or wire in different instrumentations that are not suported by sleuth ..." - possible slight rewording.